stm32loader/bootloader.py

```diff
@@ -287,6 +338,10 @@ def __init__(self, connection, verbosity=5, show_progress=None):
         self.show_progress = show_progress or ShowProgress(None)
         self.extended_erase = False
 
+    def setDataTransferAndFlashPageSize(self, device_family):
+        self.DATA_TRANSFER_SIZE = self.DATA_TRANSFER_SIZE_[device_family]
```

Comment:
By assigning to this variable, `DATA_TRANSFER_SIZE` is not longer a constant.

Please rename `DATA_TRANSFER_SIZE` to lowercase `data_transfer_size` and move its definition to `__init__()`.

Next, rename `DATA_TRANSFER_SIZE_` to `DATA_TRANSFER_SIZE (remove the trailing underscore).